introduction.md

```diff
@@ -1 +1,4 @@
 hello, the JavaScript path is helpful.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?